testData/solutions/win32.json5

```diff
@@ -316,6 +316,219 @@
+                        "options": {
+                            "focus": "CoupleNarratorCursorKeyboard",
+                            "caret": "FollowInsertion",
+                            "mouse": "InteractionMouse"
+                        }
+                    }],
+                    "EchoChars": {
+                        "transform": {
+                            "type": "fluid.transforms.condition",
+                            "false": 0,
+                            "true": 1,
+                            "conditionPath": "http://registry\\.gpii\\.net/common/keyEcho"
+                        }
+                    },
+                    "EchoWords": {
+                        "transform": {
+                            "type": "fluid.transforms.condition",
+                            "false": 0,
+                            "true": 1,
+                            "conditionPath": "http://registry\\.gpii\\.net/common/wordEcho"
+                        }
+                    }
+                }
+            },
+            "configureNoRoam": {
+                "type": "gpii.windows.registrySettingsHandler",
+                "options": {
+                    "hKey": "HKEY_CURRENT_USER",
+                    "path": "Software\\Microsoft\\Narrator\\NoRoam",
+                    "dataTypes": {
+                        "SpeechVolume": "REG_DWORD",
+                        "ShowKeyboardIntroduction": "REG_DWORD",
+                        "ShowBrowserSelection": "REG_DWORD"
+                    }
+                },
+                "supportedSettings": {
+                    "SpeechVolume": {},
+                    "ShowKeyboardIntroduction": {},
+                    "ShowBrowserSelection": {}
+                },
+                "capabilities": [
+                    "http://registry\\.gpii\\.net/common/volumeTTS"
+                ],
+                "capabilitiesTransformations": {
+                    "SpeechVolume": {
+                        "transform": {
+                            "type": "fluid.transforms.linearScale",
+                            "inputPath": "http://registry\\.gpii\\.net/common/volumeTTS",
+                            "factor": 100,
+                            "offset": 0
+                        }
+                    },
+                    "ShowKeyboardIntroduction": {
+                        "transform": {
+                            "type": "fluid.transforms.condition",
+                            "false": 0,
+                            "true": 1,
+                            "conditionPath": "http://registry\\.gpii\\.net/common/speakTutorialMessages"
+                        }
+                    },
+                    "ShowBrowserSelection": {
+                        "transform": {
+                            "type": "fluid.transforms.literalValue",
+                            "input": 0
+                        }
+                    }
+                }
+            }
```

Comment:
All the above transforms are automatcially invertible, so no `"inverseCapabilitiesTransformations"` block is needed here